README.md

```diff
@@ -10,7 +10,9 @@ We are performing data analysis with [Xarray](https://xarray.dev/),
 The Tutorial reviews the following Lessons. To get started, after having installed
 conda, create the tutorial environment by running:
```

Comment:
Maybe should add here to navigate into the `conda` subdirectory and then to run the command